daprdocs/content/en/concepts/dapr-services/placement.md

```diff
@@ -14,3 +14,59 @@ The placement service Docker container is started automatically as part of [`dap
 ## Kubernetes mode
 
 The placement service is deployed as part of `dapr init -k`, or via the Dapr Helm charts. For more information on running Dapr on Kubernetes, visit the [Kubernetes hosting page]({{< ref kubernetes >}}).
+
+## Placement tables
+
+There is an HTTP API `/placement/state` for placement service that exposes the placement table. The API is exposed on the healthz server. Since this is an unauthenticated endpoint, so it is disabled by default, you need to set `DAPR_PLACEMENT_METADATA_ENABLED` environment or `metadata-enabled` command line args to true to enable it.
+
+### Usecase:
+The placement table API can be used for retrieving the current placement table, which contains all the actors registered. This can be helpful for debugging and also allow other tools (Dashboard) to extract and present for information about Actors.

```

Comment:
```suggestion
The placement table API can be used for retrieving the current placement table, which contains all the actors registered. This can be helpful for debugging and allows tools to extract and present information about actors.

```

Reply:
Done.